mybatis-spring-boot-samples/mybatis-spring-boot-sample-annotation/pom.xml

```diff
@@ -38,11 +38,6 @@
 		</dependency>
 		<!-- test dependencies -->
 		<dependency>
-			<groupId>org.springframework.boot</groupId>
```

Comment:
Usually, projects include `spring-boot-starter-test` by default. Adding or removing `mybatis-spring-boot-starter-test` should not involve another step to add the other dependency.